docs/content/en/docs/reference/artifacts.md

```diff
@@ -3,12 +3,42 @@ title: "Artifacts"
 linkTitle: "Artifacts"
 weight: 55
 description: >
-  Artifacts associated with this release: OVAs and container images.
+  Artifacts associated with this release: OVAs and images.
 ---
 
-# OVAs
+# Bare Metal artifacts
 
-## Bottlerocket
+Artifacts for EKS Anyware Bare Metal clusters are listed below.
+If you like, you can download these images and serve them locally to speed up cluster creation.
+See descriptions of the [osImageURL]({{< relref "./clusterspec/baremetal/#osimageurl" >}}) and [`hookImagesURLPath`]({{< relref "./clusterspec/baremetal/#hookimagesurlpath" >}}) fields for details.
+
+## Ubuntu OS images for Bare Metal
+
+```
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/raw/1-20/ubuntu-v1.20.15-eks-d-1-20-17-eks-a-11-amd64.gz
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/raw/1-21/ubuntu-v1.21.13-eks-d-1-21-15-eks-a-11-amd64.gz
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/raw/1-22/ubuntu-v1.22.10-eks-d-1-22-8-eks-a-11-amd64.gz
+```
+
+## Bottlerocket OS images for Bare Metal
+
+```
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/raw/1-21/bottlerocket-v1.21.13-eks-d-1-21-15-eks-a-11-amd64.img.gz
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/raw/1-22/bottlerocket-v1.22.10-eks-d-1-22-8-eks-a-11-amd64.img.gz
+```
+
+## HookOS (kernel and initial ram disk) for Bare Metal
+
+```
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/hook/029ef8f0711579717bfd14ac5eb63cdc3e658b1d/vmlinuz-x86_64
+https://anywhere-assets.eks.amazonaws.com/releases/bundles/11/artifacts/hook/029ef8f0711579717bfd14ac5eb63cdc3e658b1d/vmlinuz-aarch64
```

Comment:
I don't think we should list aarch64 hook images because we don't build/support aarch64 OS images

Reply:
Done.